src/main/java/com/epam/lowcost/controller/UserController.java

```diff
@@ -0,0 +1,66 @@
+package com.epam.lowcost.controller;
+
+import com.epam.lowcost.model.User;
+import com.epam.lowcost.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import java.time.LocalDateTime;
+import java.util.Map;
+
+@Controller
+@RequestMapping(value = "/user")
+
+public class UserController {
+
+    @Autowired
+    UserService userService;
+
+    @GetMapping(value = "/all")
+    public String listAllUsers(Model model) {
```

Comment:
method name should be a verb, not be  a noun

Reply:
Changed to getAllUsers